packages/runtime/container-runtime/src/gc/gcSummaryStateTracker.ts

```diff
@@ -283,25 +228,14 @@ export class GCSummaryStateTracker {
 	 * Called to refresh the latest summary state. This happens when a pending summary is acked.
 	 */
 	public async refreshLatestSummary(result: IRefreshSummaryResult): Promise<void> {
```

Comment:
Want to rename this to onPendingSummaryAck?

Reply:
We can but then it would be different from rest of the system (container runtime and summarizer nodes) and lead to confusion. I would prefer we rename all the functions which is already tracked via https://dev.azure.com/fluidframework/internal/_workitems/edit/6864